cmake/HPX_SetupLibCDS.cmake

```diff
@@ -44,9 +35,14 @@ if(HPX_WITH_LIBCDS AND NOT TARGET LibCDS::cds)
     fetchcontent_populate(libcds)
     set(LIBCDS_CXX_STANDARD ${HPX_CXX_STANDARD})
     add_subdirectory(${libcds_SOURCE_DIR} ${libcds_BINARY_DIR})
+    list(TRANSFORM LIBCDS_SOURCELIST PREPEND "${libcds_SOURCE_DIR}/")
+    set(LIBCDS_SOURCE_DIR ${libcds_SOURCE_DIR})
 
-    set_target_properties(cds PROPERTIES FOLDER "Core")
-    set_target_properties(cds-s PROPERTIES FOLDER "Core")
```

Comment:
Could we leave the `FOLDER` properties in place, please (we might not need both of them now that we define our own imported target.